source/test/default_mode_test/Symbiont.test.cc

```diff
@@ -536,6 +536,68 @@ TEST_CASE("Process", "[default]") {
 
         sym.Delete();
     }
+
+    WHEN("The symbiont is free living and horizontal transmission is true") {
+      config.FREE_LIVING_SYMS(1);
+      config.SYM_HORIZ_TRANS_RES(140.0);
+      config.HORIZ_TRANS(true);
+      double int_val = 0;
+      int points = 0;
+
+      WHEN("Fls required reproduction resources is greater than -1") {
```

Comment:
Since "Fls" is an acronym, please have it in all caps to make that more clear (or spell it out)